engine/apps/auth_token/auth.py

```diff
@@ -87,6 +87,12 @@ def authenticate_credentials(self, token_string: str, request: Request) -> Tuple
         except InvalidToken:
             raise exceptions.AuthenticationFailed("Invalid token.")
 
+        # TODO: decide if we want to add this check instead of doing in plugin sync
```

Comment:
I don't think we should check these here, auth is currently checking Grafana -> OnCall is allowed.  I don't think that should fail if Grafana <- OnCall isn't working.